client/websocket_client.test.jsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import WebSocketClient from './websocket_client.jsx';
+const fakeURL = 'ws://localhostt:9001';
```

Comment:
minor nitpick: uppercase for constants
```suggestion
const FAKE_URL = 'ws://localhostt:9001';
```